js/api-augment/types.d.ts

```diff
@@ -0,0 +1,10 @@
+// Unknown is causing these to be left out of the generated types.
+// Adding them here for now.
+
+declare global {
+  type SpConsensusSlotsSlotDuration = bigint;
+  type CommonPrimitivesMsaDelegatorId = bigint;
+  type CommonPrimitivesMsaProviderId = bigint;
```

Comment:
I'm not clear on the purpose of this... but shouldn't these be `u64`, not `bigint`?

Reply:
They are bigint when others on GH have had to do it with `SpConsensusSlotsSlotDuration`, so I just kept with it, but I'll check